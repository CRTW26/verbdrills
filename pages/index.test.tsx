import '@testing-library/jest-dom'
import { render, cleanup } from '@testing-library/react'
import Index from './index'

afterEach(cleanup)

it('renders', () => {
  const { asFragment } = render(<Index />)

  expect(asFragment()).toMatchSnapshot()
})

it('should contain text "Hello World"', () => {
  const { getByTestId } = render(<Index />)

  expect(getByTestId('title')).toHaveTextContent('Hello World')
})
