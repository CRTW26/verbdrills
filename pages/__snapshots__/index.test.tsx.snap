// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <div
    class="jsx-1952444615 container"
  >
    <main
      class="jsx-1952444615"
    >
      <h1
        class="jsx-1952444615"
        data-testid="title"
      >
        Hello World
      </h1>
    </main>
  </div>
</DocumentFragment>
`;
